projects/igniteui-angular/src/lib/drop-down/drop-down.component.ts

```diff
@@ -524,6 +524,13 @@ export class IgxDropDownComponent extends IgxDropDownBaseDirective implements ID
         }
     }
 
+    /**
+     * @hidden @internal
+     */
+    public listClass() {
+        return this.items.length > 0 ? 'igx-drop-down__list' : 'igx-drop-down__list--empty';
+    }
```

Comment:
IMO this will be better to be directly bound in the dropdown template. I don't see the need to expose a whole method call just to bind a single style class.
```html
<div [class.igx-drop-down__list]="items.length" [class.igx-drop-down__list--empty]="!items.length" igxToggle [style.width]="width"
```